File: CWM.Model/Models/Expenses.cs
using CWM.Core.Models.Enums;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CWM.Core.Models
{
    public class Expenses : Base
    {
        public int TotalAmount { get; set; }
        public DateTime Date { get; set; }
        public string Description { get; set; } = string.Empty;
        public ExpensesType ExpensesType { get; set; }
        public Employee Employee { get; set; } = new();
    }
}

File: CWM.UI/cwm_desktop_mobile/lib/models/expenses.dart
import 'package:cwm_desktop_mobile/models/enums/expenses_type.dart';
import 'package:json_annotation/json_annotation.dart';

import 'employee.dart';

part 'expenses.g.dart';

@JsonSerializable()
class Expenses {
  int id;
  String description;
  DateTime date;
  Employee? employee;
  double totalAmount;
  ExpensesType expensesType;

  Expenses(this.id, this.description, this.date, this.employee,
      this.totalAmount, this.expensesType);

  factory Expenses.fromJson(Map<String, dynamic> json) =>
      _$ExpensesFromJson(json);

  Map<String, dynamic> toJson() => _$ExpensesToJson(this);
}
